### PR TITLE
[zh] improve build code-blocks diagrams

### PR DESCRIPTION
**Pull request by my-git9**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/releases/contribute/code-blocks/index.md [hunk 1]
```diff
@@ -89,7 +93,8 @@ func HelloWorld() {
 }
 {{< /text >}}
 
-你可以为代码块中的内容指定语言，以实现语法高亮。上面的例子指定了语法为 `plain`，所有代码块的渲染结果没有任何语法高亮。但是，你可以指定其语法为 `Go`，例如：
+你可以为代码块中的内容指定语言，以实现语法高亮。上面的例子指定了语法为 `plain`，
```
Comment (windsonsea) [inline content/zh/docs/releases/contribute/code-blocks/index.md:96]:
```suggestion
您可以为代码块中的内容指定语言，以实现语法高亮。上面的例子指定了语法为 `plain`，
```

File: content/zh/docs/releases/contribute/code-blocks/index.md [hunk 2]
```diff
@@ -89,7 +93,8 @@ func HelloWorld() {
...
+所有代码块的渲染结果没有任何语法高亮。但是，你可以指定其语法为 `Go`，例如：
```
Comment (windsonsea) [inline content/zh/docs/releases/contribute/code-blocks/index.md:97]:
```suggestion
所有代码块的渲染结果没有任何语法高亮。但是，您可以指定其语法为 `Go`，例如：
```
可以全局替换`你`为`您`

**Review by windsonsea — COMMENTED** [review]
note the lint error:

```
./zh/docs/releases/contribute/diagrams/index.md:10: MD009 Trailing spaces
```

File: content/zh/docs/releases/contribute/build/index.md [hunk 3]
```diff
@@ -21,19 +25,23 @@ keywords: [contribute, serve, Docker, Hugo, build]
 $ make serve
 {{< /text >}}
 
-如果您的修改没有出现编译错误，这个命令会构建这个站点，并且启动一个本地 web 服务来运行这个站点。通过在浏览器中输入 `http://localhost:1313` 来浏览本地构建的站点。
+如果您的修改没有出现编译错误，这个命令会构建这个站点，并且启动一个本地 web
```
Comment (wilsonwu) [inline content/zh/docs/releases/contribute/build/index.md:28]:
```suggestion
如果您的修改没有出现编译错误，这个命令会构建这个站点，并且启动一个本地 Web
```

File: content/zh/docs/releases/contribute/build/index.md [hunk 4]
```diff
@@ -21,19 +25,23 @@ keywords: [contribute, serve, Docker, Hugo, build]
...
+如果您要从远程服务器上构建和预览，可以使用 `ISTIO_SERVE_DOMAIN`
+来设置这个服务器的 IP 地址或者 DNS 域名，例如：
 
 {{< text bash >}}
 $ make ISTIO_SERVE_DOMAIN=192.168.7.105 serve
 {{< /text >}}
 
-这个例子是在 `192.168.7.105` 这个远程服务器上构建站点并且启动一个 web 服务器。像前面一样，可以通过 `http://192.168.7.105:1313` 这个地址来访问这个 web 服务器。
+这个例子是在 `192.168.7.105` 这个远程服务器上构建站点并且启动一个 web 服务器。
+像前面一样，可以通过 `http://192.168.7.105:1313` 这个地址来访问这个 web 服务器。
```
Comment (wilsonwu) [inline content/zh/docs/releases/contribute/build/index.md:39]:
```suggestion
这个例子是在 `192.168.7.105` 这个远程服务器上构建站点并且启动一个 Web 服务器。
像前面一样，可以通过 `http://192.168.7.105:1313` 这个地址来访问这个 Web 服务器。
```

File: content/zh/docs/releases/contribute/code-blocks/index.md [hunk 5]
```diff
@@ -127,7 +132,9 @@ Istio 代码块目前支持下面这些语言的语法高亮：
 - `docker`
 - `bash`
 
-默认情况下，CLI 命名的输出结果会按照 plain 文本进行渲染，即没有语法高亮。如果你想为输出添加语法高亮，你可以在短代码中指定语法。在 Istio 中，最常见的例子就是 YAML 和 JSON 的输出结果，例如：
+默认情况下，CLI 命名的输出结果会按照 plain 文本进行渲染，即没有语法高亮。
+如果你想为输出添加语法高亮，你可以在短代码中指定语法。在 Istio 中，最常见的例子就是
```
Comment (wilsonwu) [inline content/zh/docs/releases/contribute/code-blocks/index.md]:
```suggestion
如果您想为输出添加语法高亮，您可以在短代码中指定语法。在 Istio 中，最常见的例子就是
```

File: content/zh/docs/releases/contribute/code-blocks/index.md [hunk 6]
```diff
@@ -169,9 +177,10 @@ $ kubectl -n istio-system logs $(kubectl -n istio-system get pods -l istio-mixer
 
 通过设置 `syntax=` 字段的值，指定内容的语言，可以获得语法高亮的渲染。
 
-### 通过 URL 从外部资源导入代码{#import-code-from-an-external-source-through-a-URL}
+### 通过 URL 从外部资源导入代码  {#import-code-from-an-external-source-through-a-URL}
 
-类似的，你可以从互联网动态的导入内容。使用 `url` 属性指定资源。下面的例子导入了同一个文件，但它是通过 URL 导入的：
+类似的，你可以从互联网动态的导入内容。使用 `url` 属性指定资源。
```
Comment (wilsonwu) [inline content/zh/docs/releases/contribute/code-blocks/index.md]:
```suggestion
类似的，您可以从互联网动态的导入内容。使用 `url` 属性指定资源。
```

**Review by wilsonwu — APPROVED** [review]
Good job, LGTM for the content, only need lint fix.

File: content/zh/docs/releases/contribute/diagrams/index.md [hunk 7]
```diff
@@ -7,9 +7,10 @@ keywords: [contribute,diagram,documentation,guide]
 
 欢迎学习 Istio 图表指南！
 
-本指南通过一个 [SVG 文件](./diagram-guidelines.svg)或者 [Google Drawings 文件](https://docs.google.com/drawings/d/1f3NyutAQIDOA8ojGNyMA5JAJllDShZGQAFfdD01XdSc/edit)使您可以轻松的复用 shape 和 style。
-通过本指南，您可以使用任何矢量图绘制工具（例如，Google Drawings、Inkscape、Illustrator）为 Istio 站点创建 SVG 图表。
-请保持图表中的文本处于可编辑状态。
+本指南通过一个 [SVG 文件](./diagram-guidelines.svg)或者
+[Google Drawings 文件](https://docs.google.com/drawings/d/1f3NyutAQIDOA8ojGNyMA5JAJllDShZGQAFfdD01XdSc/edit)使您可以轻松的复用
+hape 和 style。通过本指南，您可以使用任何矢量图绘制工具（例如，Google Drawings、Inkscape、Illustrator）
```
Comment (wilsonwu) [inline content/zh/docs/releases/contribute/diagrams/index.md:12]:
```suggestion
shape 和 style。通过本指南，您可以使用任何矢量图绘制工具（例如，Google Drawings、Inkscape、Illustrator）
```

**Review by wilsonwu — CHANGES_REQUESTED** [review]
Miss a letter 's'

**Review by windsonsea — APPROVED** [review]
Thanks. LGTM

**Pull request merged**